src/common/hook/useQueue/utils/publish.tsx

```diff
@@ -10,31 +10,26 @@ import { getQueueNumber } from "../../../API/storageAPI";
 import { encodeQrCode, getDocumentNetwork, getDataV3 } from "../../../utils";
 import { Signer } from "ethers";
 import { publishDnsDidVerifiableDocumentJob } from "../../../../services/publishing";
+import { ChainInfo } from "../../../../constants/chainInfo";
 
-interface NetworkUrl {
-  homestead: string;
-  ropsten: string;
-  rinkeby: string;
-}
-
-const getReservedStorageUrl = async (
-  documentStorage: DocumentStorage,
-  network: "homestead" | "ropsten" | "rinkeby"
-): Promise<ActionsUrlObject> => {
+const getReservedStorageUrl = async (documentStorage: DocumentStorage, network: string): Promise<ActionsUrlObject> => {
```

Comment:
can import the `Network` type from `types/index.d.ts`

Reply:
same reason as [this](https://github.com/TradeTrust/document-creator-website/pull/264#discussion_r996606758). if so, after https://github.com/TradeTrust/document-creator-website/pull/264 then i import from there then